swarm/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
   See [PR 4715](https://github.com/libp2p/rust-libp2p/pull/4715).
 - Log `PeerId` of `Swarm` even when constructed with new `SwarmBuilder`.
   See [PR 4671](https://github.com/libp2p/rust-libp2p/pull/4671).
+- Remove private `p2p_addr` function and replace it with the newly added `with_p2p` on `Multiaddr`.
```

Comment:
@thomaseizinger  I did not know if you wanted a Changelog for a private function. I still created it but I can of course remove it if you think it is not necessary.

Reply:
The changelogs are for users only. Thus no need to document a change to a private function.